thoth/common/openshift.py

```diff
@@ -1374,6 +1405,52 @@ def schedule_graph_sync_package_extract(
             document_id, namespace, template_name="graph-sync-job-package-extract"
         )
 
+    def run_graph_sync_multiple(
+        self,
+        *,
+        only_solver_documents: bool = False,
+        only_analysis_documents: bool = False,
+        only_inspection_documents: bool = False,
+        only_adviser_documents: bool = False,
+        only_provenance_checker_documents: bool = False,
+        only_dependency_monkey_documents: bool = False,
+        namespace: str = None,
+        template: dict = None,
+        job_id: str = None,
+        template_name: str = None,
+    ) -> str:
+        """Run the given graph sync to sync multiple documents into graph database."""
+        if not namespace:
+            raise ValueError(
+                "Unable to run graph sync without namespace being specified"
+            )
+
+        if not template and not template_name:
+            raise ValueError(
+                "At least one of `template` and `template_name` should be specified"
+            )
+
+        template = template or self.get_graph_sync_template(template_name)
+        self.set_template_parameters(
+            template,
+            THOTH_JOB_ID=job_id or self._generate_id(template_name),
+            THOTH_ONLY_SOLVER_DOCUMENTS=int(only_solver_documents),
+            THOTH_ONLY_ANALYSIS_DOCUMENTS=int(only_analysis_documents),
+            THOTH_ONLY_INSPECTION_DOCUMENTS=int(only_inspection_documents),
+            THOTH_ONLY_ADVISER_DOCUMENTS=int(only_adviser_documents),
+            THOTH_ONLY_PROVENANCE_CHECKER_DOCUMENTS=int(only_provenance_checker_documents),
+            THOTH_ONLY_DEPENDENCY_MONKEY_DOCUMENTS=int(only_dependency_monkey_documents),
+        )
+        template = self.oc_process(namespace, template)
+        graph_sync = template["objects"][0]
+
+        response = self.ocp_client.resources.get(
+            api_version=graph_sync["apiVersion"], kind=graph_sync["kind"]
+        ).create(body=graph_sync, namespace=namespace)
+
+        _LOGGER.info("Scheduled new graog sync multiple with id %r", response.metadata.name)
```

Comment:
there a typo here for graph. :+1:

Reply:
Nice catch! Should be fixed now.

Reply:
I thought it was a test of my review skills :see_no_evil: